src/ZodError.ts

```diff
@@ -360,6 +360,10 @@ export const defaultErrorMap = (
         message = `Number must be greater than ${
           issue.inclusive ? `or equal to ` : ``
         }${issue.minimum}`;
+      else if (issue.type === "date")
+        message = `Date must be greater than or equal to ${new Date(
```

Comment:
Should use issue.inclusive to decide whether the "or equal to" should be added, as is done for other types.